cmd/report.go

```diff
@@ -53,15 +53,12 @@ var (
 	}
 	reportCookbooksCmd = &cobra.Command{
 		Use:   "cookbooks",
-		Short: "Generates a cookbook oriented report",
+		Short: "Generates a cookbook-oriented report",
 		Args:  cobra.NoArgs,
-		Long: `Generates cookbook oriented reports containing details about the number of
-violations each cookbook has, which violations can be are auto-corrected and
-the number of nodes using each cookbook.
+		Long: `Generates a cookbook-oriented report containing details about the
+upgrade compatibility errors and node cookbook usage.
 
-These reports could take a long time to run depending on the number of cookbooks
```

Comment:
This seemed to be a lot of words that wouldn't be read - we don't need to explain why we're doing a thing, only that we are doing it.